go/gas-oracle/flags/flags.go

```diff
@@ -89,6 +89,12 @@ var (
 		Usage:  "length of epochs in seconds",
 		EnvVar: "GAS_PRICE_ORACLE_EPOCH_LENGTH_SECONDS",
 	}
+	L1BaseFeeEpochLengthSecondsFlag = cli.Uint64Flag{
```

Comment:
But what do you mean it'll be a breaking change to make this a duration? It's a new env flag so there shouldn't be any users to break.

Reply:
Ah sorry to clarify the flag directly above it is not a duration flag so i didn't make this one a duration flag